File: Des.web/Desenvolvimento web/python/algoritmo2.py
# construa um algotimo que receba um numero inteiro do usuario, calcule seu fatorial da entrada,
# exiba o passo a passo da operação e mostre o resultado

num = int(input("Digite um número: "))

fatorial = 1

for i in range(num, 0, -1):
    fatorial = fatorial * i
    print("%d! = %d" %(i, fatorial))

print("Resultado: %d" %fatorial)

File: Des.web/Desenvolvimento web/php/algoritmo2.php
<!-- Exemplo:
	- Entrada: 5
	- Saída: 
	5! = 5 * 4 * 3 * 2 * 1 = 120

    # 1! = 1 * 0!
	# 2! = 2 * 1!
	# 3! = 3 * 2!
	# 4! = 4 * 3!
	# 5! = 5 * 4! -->

<?php	
	$num = $_POST['num'];
	$fatorial = 1;
	$tmp = $num;
	
	for($i = 0; $i < $num; $i++) {
		echo $tmp . '! = ' . $tmp . ' * ' . ($tmp - 1) . ' = ';
		$fatorial *= $tmp;
		$tmp--;
		echo $fatorial . '<br>';
	}
	
	echo 'O fatorial do número ' . $num . ' é: ' . $fatorial;
?>